packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -5,8 +5,8 @@
 //!
 //! The format implementation (especially the [`Debug`] implementation),
 //! can be easily extended using hooks.
-//! Hooks are functions of the signature `Fn(&T, &mut HookContext<T>) -> Emit`, they provide an
-//! easy and ergonomic way to partially modify the format and enable the output of types that are
+//! Hooks are functions of the signature `Fn(&T, &mut HookContext<T>) -> Diagnostics`, they provide
```

Comment:
incorrect docs, should be `Vec<Emit>`

Reply:
fixed in [4892398](https://github.com/hashintel/hash/pull/976/commits/4892398d338cb681fe7bbd5443b584fdc6383fad)